<launch>    
	<!-- Arguments -->
	<arg name="filename" default="$(find occupancy_map_visualizer)/samples/gridmap3d.og3" />

    <!-- Parameters -->
    <param name="fixed_frame_id"		value="map" />
    <param name="gridmap3d_filename" 	value="$(arg filename)" />

    <!-- Run Rviz -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find occupancy_map_visualizer)/rviz/gridmap3d.rviz" />

    <!-- GridMap3D Visualization -->
    <node pkg="occupancy_map_visualizer"	type="gridmap3d_visualizer_node"	name="gridmap3d_visualizer_node"	output="screen"/>
</launch>
